model_tree.js

```diff
@@ -0,0 +1,250 @@
+module.exports = {
```

Comment:
Do we want this to be committed and pretty formatted or autogenerated in post-install?

Reply:
I think autogenerated post-install should would be a good solution.